stream/address/import.js

```diff
@@ -20,52 +19,30 @@ function streamFactory(db, done){
   // create a new stream
   return through.obj({ highWaterMark: 2 }, function( batch, _, next ){
 
-    // run serially so we can use transactions
-    db.serialize(function() {
+    // start transaction
+    db.transaction(() => {
 
-      // start transaction
-      db.run('BEGIN TRANSACTION', function(err){
+      // import batch
```

Comment:
same re: comment - obviously the next line is iterating over "batch" - this comment doesn't help me really understand what's going on inside of it, it just duplicates the line below